io/post_fv3.F90

```diff
@@ -4609,12 +4758,18 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
           enddo
         enddo ! do loop for l
 
-        l=lm
-        do j=jsta,jend
+        do l=1,lm
```

Comment:
do j=jsta,jend
           do i=ista,iend
              do l=1,lm

Reply:
Those lines are already in the file, in other loops. Look for `! chmical field computation` in the file. (Typo is in the original comment.)

That page has instructions for the "suggested change" feature:

- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request